doc/getting-started/Kotlin-Versions.md

```diff
@@ -0,0 +1,29 @@
+## Kotlin versions and the Gradle Kotlin DSL 
+
+Gradle ships with Kotlin DSL support which uses embedded Kotlin compiler, plus fixed Kotlin `stdlib` and `reflect`
+dependencies. For example, Gradle 4.3 ships with Kotlin 1.1.51, as well as `stdlib` and `reflect` with the same version.
+The `kotlin` package from those modules is visible through the Gradle classpath.
+
+The [compatibility guarantees](https://kotlinlang.org/docs/reference/compatibility.html) provided by Kotlin apply for
+both backward and forward compatibility.
+
+### Backward compatibility
+
+Our approach is to only do backwards-breaking Kotlin upgrades on a major Gradle release. We will always clearly document
+which Kotlin version we ship and announce upgrade plans before a major release. 
+
+> Until the Gradle Kotlin DSL v1.0 release our policy will be to ship with the latest stable Kotlin version available
```

Comment:
⭕️  Maybe _Until the release of Gradle Kotlin DSL v1.0 our policy..._ ?

Reply:
👍 
Done